native/package.json

```diff
@@ -75,7 +75,7 @@
     "query-string": "^7.1.1",
     "react": "18.2.0",
     "react-i18next": "^11.18.0",
-    "react-native": "0.70.6",
+    "react-native": "0.70.9",
```

Comment:
Any specific reason why we upgrade react native? Normally this should just be done in a real issue. Did you check possibly necessary manual upgrading steps?